dev/CommonStyles/ComboBox_themeresources.xaml

```diff
@@ -356,8 +356,12 @@
     <x:Int32 x:Key="ComboBoxPopupMaxNumberOfItems">15</x:Int32>
     <x:Int32 x:Key="ComboBoxPopupMaxNumberOfItemsThatCanBeShownOnOneSide">7</x:Int32>
 
+    <Thickness x:Key="ComboBoxPadding">12,5,0,7</Thickness>
```

Comment:
Your PR description only talks about TreeView and NavigationView. So, ComboBox changes too?

Reply:
I renamed the title